src/cli/javascript/js_request.rs

```diff
@@ -50,12 +50,9 @@ impl TryFrom<reqwest::Request> for JsRequest {
                 )
             })
             .collect::<BTreeMap<String, String>>();
-        let body = req.body().map(|body| {
-            let bytes = body.as_bytes().unwrap_or_default();
-            Bytes::from_iter(bytes.to_vec())
-        });
 
-        Ok(JsRequest { url, method, headers, body })
+        // NOTE: We don't pass body to worker for performance reasons
+        Ok(JsRequest { url, method, headers, body: None })
```

Comment:
We actually need `body` then? From my findings, the only place we actually use it is on tests.

Reply:
Request from rust will not contain body. Request from JS can contain body.